app/backend/submissions/serializers.py

```diff
@@ -486,6 +487,16 @@ class Meta:
             'internal_comments',
             'alternative_specs_submitted',
             'decommission_description_set',
+            'avi',
```

Comment:
Is this the aquifer vulnerability index? Either way - this seems to be the only acronym we have. Should we not be consistent and call things what they are?